shell/src/main/java/alluxio/cli/job/command/GetCmdStatusCommand.java

```diff
@@ -53,43 +62,60 @@ public String getCommandName() {
     return "getCmdStatus";
   }
 
+  @Override
+  public Options getOptions() {
+    return new Options().addOption(VERBOSE_OPTION);
+  }
+
   @Override
   public void validateArgs(CommandLine cl) throws InvalidArgumentException {
     CommandUtils.checkNumOfArgsEquals(this, cl, 1);
   }
 
   @Override
   public String getUsage() {
-    return "getCmdStatus <jobControlId>";
+    return "getCmdStatus [-v] <jobControlId>";
   }
 
   @Override
   public String getDescription() {
-    return "Get the status information for a distributed command.";
+    return "Get the status information for a distributed command. "
+            + "Use -v flag to display the status in detail.";
   }
 
   @Override
   public int run(CommandLine cl) throws AlluxioException, IOException {
     String[] args = cl.getArgs();
     long jobControlId = Long.parseLong(args[0]);
-    Set<String> failedFiles = Sets.newHashSet();
-    List<String> completedFiles = Lists.newArrayList();
-
     try (CloseableResource<JobMasterClient> client =
                  JobContext.create(mFsContext.getClusterConf(),
                                  mFsContext.getClientContext().getUserState())
                          .acquireMasterClientResource()) {
-      DistributedCommandUtil.getDetailedCmdStatus(
-              jobControlId, client.get(), failedFiles, completedFiles);
-      if (!failedFiles.isEmpty()) {
-        System.out.println("Failed files are:");
-        failedFiles.forEach(System.out::println);
+      if (cl.hasOption("v")) {
+        Set<String> failedFiles = Sets.newHashSet();
+        List<String> completedFiles = Lists.newArrayList();
+        DistributedCommandUtil.getDetailedCmdStatus(
+                jobControlId, client.get(), failedFiles, completedFiles);
+        if (!failedFiles.isEmpty()) {
+          System.out.println("Failed files are:");
+          failedFiles.forEach(System.out::println);
+        }
+      } else {
+        Status status = client.get().getCmdStatus(jobControlId);
+        System.out.println(status);
       }
     } catch (Exception e) {
-      LOG.error("Failed to get detailed status of the command", e);
-      System.out.println(String.format("Unable to get detailed information for command %s."
-                      + " Please retry using `getCmdStatus` to check command detailed status,",
-              jobControlId));
+      if (cl.hasOption("v")) {
+        LOG.error("Failed to get detailed status of the command", e);
+        System.out.format("Unable to get detailed information for command %s.%n"
```

Comment:
It seems jobControlId is the only parameter followed but you have %s, %n and %n

Reply:
corrected.